sass/themes/sr/2018/components/teaser-block/_teaser-block.scss

```diff
@@ -2,6 +2,36 @@
   padding: 20px;
   text-align: initial;
 }
-.teaser-block--bg-image .teaser-block__body {
-  padding: 0;
+
+.teaser-block--bg-image {
+  .teaser-block__image:after {
+    width: 100%;
+    height: 100%;
+    content: "";
+    position: absolute;
+    top: 0;
+    left:0;
+    transition: background-color 0.2s linear;
+    // Set this up as totally transparent to fade-in on hover
+    background-color: rgba(0,0,0,0);
+  }
+
+  .teaser-block__body {
+    /* Increase padding-top to extend gradient 'reach';
+     * provides better accessibility against the first line of text */
+    padding: 80px 30px 40px;
+    margin: 0;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    top: auto;
+    height: auto;
+    background-image: linear-gradient(transparent, #29022c);
```

Comment:
hey we're using comic relief dark purple instead of the sr18 dark blue for this?

Reply:
Well-spotted; I just need Jimmy Squizzles to pick me out some final colours